### PR TITLE
use facebook_business_sdk instead of homegrown rest client

### PR DESCRIPTION
**Pull request by sloev**
# Description of change
(write a short description or paste a link to JIRA)

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: tap_facebook/streams.py [hunk 1]
```diff
@@ -42,26 +43,91 @@ def process_account(
...
+            "unique_inline_link_clicks"
+        ]
         with singer.metrics.record_counter(tap_stream_id) as counter:
+
+            if not start_date:
+                raise ValueError("client: start_date is required")
+
+            since = utils.parse_date(start_date)
+
+            until = date.today() + timedelta(days=-1)
```
Comment (przywartya) [inline tap_facebook/streams.py:78]: more readable: `- timedelta(days=1)`? 😄

File: tap_facebook/streams.py [hunk 2]
```diff
@@ -42,26 +43,91 @@ def process_account(
...
+
+            increment = 1
```
Comment (przywartya) [inline tap_facebook/streams.py]: This increment is never modified to more than `1`, or at least I don't see where is it modified

Comment (przywartya) [reply tap_facebook/streams.py]: So the timeranges are always 1 day difference

**Review by przywartya — APPROVED** [review]
One comment, all else is good :)

File: tap_facebook/streams.py (hunk 1 above)
Comment (sloev) [reply tap_facebook/streams.py:78]: i agree :-)

**Pull request merged**